modelforge/tests/test_models.py

```diff
@@ -37,20 +47,39 @@ def test_model_factory(model_name, simulation_environment):
     )
     from modelforge.train.training import TrainingAdapter
 
+    # read default parameters
+    from modelforge.train.training import return_toml_config
+
+    config = return_toml_config(
+        f"modelforge/tests/data/potential_defaults/{model_name.lower()}_defaults.toml"
```

Comment:
This will only work if you run pytest from modelforge's root directory.  We should update this to get the full path using resources (I will push a change for this) and also make sure that .toml files are included in the setup file for pip so they are copied over to the egg.

Reply:
thanks! that is a good point!